workflow/setup_expt.py

```diff
@@ -73,41 +64,36 @@ def _update_defaults(dict_in: dict) -> dict:
     # _update_defaults replaces any keys/values in defaults with matching keys in base
     yaml_dict = _update_defaults(yaml_dict)
 
-    # Override the YAML defaults with the host-specific capabilities
-    # First update config.base
-    edit_baseconfig(host, inputs, yaml_dict)
-
-    # Update stage config
-    stage_dict = {
-        "@ICSDIR@": inputs.icsdir
-    }
-    host_dict = get_template_dict(host.info)
-    stage_dict = dict(stage_dict, **host_dict)
-    stage_input = f'{inputs.configdir}/config.stage_ic'
-    stage_output = f'{inputs.expdir}/{inputs.pslot}/config.stage_ic'
-    edit_config(stage_input, stage_output, host_dict, stage_dict)
-
-    # Loop over other configs and update them with defaults
-    for cfg in yaml_dict.keys():
-        if cfg == 'base':
-            continue
-        cfg_file = f'{inputs.expdir}/{inputs.pslot}/config.{cfg}'
-        cfg_dict = get_template_dict(yaml_dict[cfg])
-        edit_config(cfg_file, cfg_file, host_dict, cfg_dict)
+    # Need to map inputs_dict keys to keys used in configs
+    inputs_dict_remapped = map_inputs_to_configs(inputs)
+
+    # Combine host.info and inputs_dict into a single dict
+    combined_dict = AttrDict(host.info, **inputs_dict_remapped)
```

Comment:
I think this should be reversed.  `host.info` should override the options set in the CI options in, e.g. `ci/cases/yamls/gfs_metp_ci.yaml`:

```
    combined_dict = AttrDict(**inputs_dict_remapped, host.info)
```

Reply:
The above suggestion did not work on Gaea-C6.  Still looking.